datahub-ranger-plugin/build.gradle

```diff
@@ -24,8 +24,11 @@ compileJava {
 }
 
 dependencies {
-    implementation 'org.apache.ranger:ranger-plugins-common:2.3.0'
-    compile 'org.apache.logging.log4j:log4j-1.2-api:2.17.1'
+    implementation(externalDependency.rangerCommons) {
+        exclude group: "org.apache.htrace", module: "htrace-core4"
+    }
+    implementation externalDependency.hadoopCommon3
+    implementation 'org.apache.logging.log4j:log4j-1.2-api:2.17.1'
```

Comment:
Let's push this into an external dependency definition too with the others unless there's a strong reason to keep it separate.